src/components/errors-component.js

```diff
@@ -15,6 +15,11 @@ import isValid from '../form/is-valid';
 import resolveModel from '../utils/resolve-model';
```

Comment:
Notable changes include:

1. Added the defaultStrategy
2. Wrapped the majority of the contents in function createErrorsClass(s = defaultStrategy) {} block
3. Converted use of get, getForm and actions to be retrieved from the strategy